package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "posthog-js",
-    "version": "1.4.3",
+    "version": "1.4.4",
```

Comment:
I think we should bump versions outside PRs... This is hard to merge now and should be rebased